### PR TITLE
openstack Check to see if firewall rule has destination defined

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/terraform/checks/resource/openstack/example_FirewallRuleSetDestinationIP/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+resource "openstack_fw_rule_v1" "pass" {
+  name             = "my_small_world"
+  description      = "let anyone in"
+  action           = "allow"
+  protocol         = "tcp"
+  destination_port = "22"
+  enabled          = "true"
+  destination_ip_address = "10.0.0.1"
```
Comment (gruebel) [inline tests/terraform/checks/resource/openstack/example_FirewallRuleSetDestinationIP/main.tf:18]: what happens, if I put in `0.0.0.0` is it then open to everyone? The documentation also states that you can a subnet os this `0.0.0.0/0` could be possible?

Comment (JamesWoolfenden) [reply tests/terraform/checks/resource/openstack/example_FirewallRuleSetDestinationIP/main.tf:28]: good catch. fixed.

File: checkov/terraform/checks/resource/openstack/FirewallRuleSetDestinationIP.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        supported_resources = ['openstack_fw_rule_v1']
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources,
+                         missing_attribute_result=CheckResult.FAILED)
+
+    def get_inspected_key(self) -> str:
+        return 'destination_ip_address'
+
+    def get_forbidden_values(self) -> str:
+        return ["0.0.0.0/0", "0.0.0.0"]
```
Comment (gruebel) [inline checkov/terraform/checks/resource/openstack/FirewallRuleSetDestinationIP.py:19]:
```suggestion
        return ["0.0.0.0/0", "0.0.0.0"]  # nosec
```
I think that's all you need to fix the `bandit` issue, but please test it 😄

Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/openstack/FirewallRuleSetDestinationIP.py:19]: thats just a bandit suppression?

Comment (gruebel) [reply checkov/terraform/checks/resource/openstack/FirewallRuleSetDestinationIP.py:19]: yes correct
https://github.com/PyCQA/bandit#exclusions

**Review by gruebel — APPROVED** [review]
👍

**Pull request merged**